src/test/java/org/jenkinsci/plugins/badge/ParameterResolverTest.java

```diff
@@ -1,14 +1,16 @@
 package org.jenkinsci.plugins.badge;
 
 import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 import hudson.model.Actionable;
+import org.junit.Test;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.mockito.Mockito;
 
-class ParameterResolverTest {
+public class ParameterResolverTest {
```

Comment:
JUnit 5 tests do not require that the class is public.  They [suggest](https://junit.org/junit5/docs/snapshot/user-guide/#writing-tests-classes-and-methods) that the classes generally should not be public.
```suggestion
class ParameterResolverTest {
```

Reply:
JUnit 5 does not use public signature